src/v2/view-builder/views/IdentifierView.js

```diff
@@ -94,18 +94,6 @@ const Body = BaseForm.extend({
     }
   },
 
-  showMessages() {
-    const messagesObj = this.options.appState.get('messages');
-    if (messagesObj?.value.length
-        && this.options.appState.get('currentFormName') === 'identify') {
-      const displayMessageObj = messagesObj.value[0];
-      const messageCallout = createCallout({
```

Comment:
before for this view we were rendering a callout all the time, with the new changes it will either render a callout or just a `<p>`, depending if the `messagesObj.value[0].class` is defined. This is why I was asking in the BaseForm for some screenshots, to better see what is changing

Reply:
I have to take a leap of faith here BUT it's a part of fix.. 

All the messages tagged as `ERROR` or `WARN` should have a callout.. and the messages that are not tagged or have a class `INFO` then it should be shown as `plain text message`... 

If an IDX api not adding a `class` that it expect widget to render as error.. then we should fix the API.

I sifted through `testcafes` i don't see any example for `IdentifierView` that was showing `error callout` for messages when `class` was undefined

Reply:
Ok, we can take the leap of faith given you said the API messages must come with a `class` value